src/tests/suite/daos_test_common.c

```diff
@@ -899,8 +902,19 @@ daos_kill_server(test_arg_t *arg, const uuid_t pool_uuid,
 		      "disabled, svc->rl_nr %d)!\n", rank, arg->srv_ntgts,
 		       arg->srv_disabled_ntgts - 1, svc->rl_nr);
 
-	rc = daos_mgmt_svc_rip(grp, rank, true, NULL);
-	assert_int_equal(rc, 0);
+	fp = popen("which dmg", "r");
```

Comment:
what is the purpose of this?   Either dmg is in the path or it isn't.   Why do we need to resolve the full path before creating the command?  I would just remove this block and use dmg.   Regardless, I don't think calling which is the right strategy here.

Reply:
It is a not clean handling - several tests failed with  system(dmg) return code 0x100, likely is not able to locate dmg. After passing the full path, the test worked.

Not clear how dmg path is define in CI test in this daos_test C code, I know Avocado code set it in python using .build_vars.json, Wondering we need to do something similar in C code.

Reply:
This was the test wihtou "which dmg" rebase 4 days ago that had more failures with 0x100 return. Let me re-test with out full path, hopefully the failures were caused by other issues which is addressed.
https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-2620/13/tests

Reply:
It passed CI functional test without full path.